windowsSetup.md

```diff
@@ -0,0 +1,19 @@
+# Set up caMicroscope for Windows 10 Home
+
+1. Install Windows Subsystem for Linux (WSL2) from the link given here: <a href = "https://docs.microsoft.com/en-us/windows/wsl/install-win10#manual-installation-steps">WSL2</a>, **FOLLOW STEPS 1 to 5 ONLY (Step 6 - Install your Linux distribution of choice IS NOT REQUIRED)** 
+2. After completing all the five steps, <a href = "">Install Docker Desktop for Windows</a>, click on the "Get Docker" button on the right and run the "Docker Desktop Installer" to complete the installation.
+3. Now, select any folder (say E:/) and open the command prompt.
+4. Clone the three repositories: caMicroscope, Distro and Caracal using <code>git clone</code> commands given below:<br>
+    <code>git clone https://github.com/camicroscope/caMicroscope.git</code><br>
+    <code>git clone https://github.com/camicroscope/Caracal.git</code><br>
+    <code>git clone https://github.com/camicroscope/Distro.git</code><br>
+
+5. Open the "caMicroscope.yml" file which can be found in the Distro folder, "Distro/caMicroscope.yml".
+6. Add <code>DISABLE_SEC: "true"</code> as shown below, adding this line will help us get rid of JsonWebTokenError.
```

Comment:
This is part of what we use develop.yml for.

Reply:
Yes, you are right. Can you please tell me the difference between DISABLE_SEC and ALLOW_PUBLIC? Also, I am new to Docker, so can you tell me the main difference between running caMicroscope.yml file and develop.yml file.

Reply:
ALLOW_PUBLIC keeps login checks, but allows someone who has not been signed up to access public slides. See https://github.com/camicroscope/Caracal/blob/2a5be469a1e61fece67cafd2d5bb18c8510b2792/handlers/userFunction.js#L9
DISABLE_SEC ignores all auth checks.

Reply:
develop.yml and caMicrosope.yml use different branches and have some config changes to make development easier, and setting DISABLE_SEC is one of those.

Reply:
Thank you.